<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifySouvenirs</class>
 <widget class="QDialog" name="ModifySouvenirs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modify Souvenirs</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>781</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Add Souvenir</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Team Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>207</x>
       <y>120</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Souvenir Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Souvenir Price:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="team_comboBox">
     <property name="geometry">
      <rect>
       <x>322</x>
       <y>70</y>
       <width>201</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="souvName_lineEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>120</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Souvenir Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="souvPrice_lineEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>170</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Souvenir Price</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addSouvButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Souvenir</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Modify Souvenir</string>
    </attribute>
    <widget class="QPushButton" name="modifySouvenirButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>200</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Modify Souvenir</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="souvPrice_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>150</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Souvenir Price</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Price:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="team2_comboBox">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Team Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Souvenir:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="souv2_lineEdit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>110</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Souvenir Name</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Delete Souvenir</string>
    </attribute>
    <widget class="QPushButton" name="deleteSouvenir_pushButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>220</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Souvenir</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>58</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Team:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Souvenir:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>211</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="team3_comboBox"/>
      </item>
      <item>
       <widget class="QLineEdit" name="souv_lineEdit">
        <property name="placeholderText">
         <string>Souvenir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
